stylesheets/commons/MatchTicker.less

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+Template(s): Match Ticker
+Author(s): Nadox
+*******************************************************************************/
+.new-match-style {
+	.match-section-header {
+		display: flex;
+		flex-direction: column;
+		padding: 1rem;
+		gap: 0.5em;
+	}
+
+	.match {
+		border-bottom: 1px solid var( --table-border-color, var( --clr-border, #bbbbbb ) ) !important;
+		padding: 0.75rem 1rem;
+		font-size: 0.875rem;
+
+		.match-scoreboard {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			gap: 2rem;
+			margin-bottom: 0.5rem;
+
+			> div {
+				flex: 1;
+			}
+
+			.team-left {
+				display: flex;
+				justify-content: right;
+			}
+
+			.versus {
+				flex: 0 1 auto;
+				display: flex;
+				flex-direction: column;
+				justify-content: center;
+				align-items: center;
+
+				.versus-lower {
+					font-size: 0.75rem;
+					color: var( --clr-secondary );
+				}
+			}
+		}
+
+		.match-details {
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+			padding: 0.5rem 0.75rem;
+			border-radius: 0.5rem;
+			font-size: 0.75rem;
+
+			.theme--light & {
+				background-color: #0000000a;
+			}
+
+			.theme--dark & {
+				background-color: #ffffff0a;
+			}
+
+			.match-streams {
+				display: flex;
+				align-items: center;
+				justify-content: center;
+				gap: 0.25rem;
+			}
+
+			.timer-hidden {
```

Comment:
If this is a modifier class, you'd want it write it as timer--hidden.

Reply:
Removed it as it should be in the other MR